### PR TITLE
Initial commit on OpenCV-Python Cheat Sheet

### PR DESCRIPTION
**Pull request by anselal**
I have created an IA page at https://duck.co/ia/view/opencv_python_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@anselal Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=opencv python cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/opencv_python_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=opencv python cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/opencv_python_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=opencv python cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/opencv_python_cheat_sheet_mobile.png)

**Comment by edgesince84** [comment]
@anselal,  thanks again for creating the OpenCV-Python Cheat Sheet! You’ve done a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Comment by moollaza** [comment]
@anselal Thanks! This LGTM :+1:

File: share/goodie/cheat_sheets/json/opencv-python.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                "val":"Convert the Image to the L*a*b* Color Space"
+            },
+            {
+                "key":"gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)",
+                "val":"Convert the Image to Grayscale"
+            }
+        ],
+        "Rotation": [
+            {
+                "key": "degrees = 45\n\n# calculate the center of the image\n(h, w) = img.shape\\[:2\\]\n(cX, cY) = (w / 2, h / 2)\n\nM = cv2.getRotationMatrix2D((cX, cY), degrees, 1.0)\nrotated = cv2.warpAffine(image, M, (w, h))",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/opencv-python.json]: This one looks a little odd given how lengthy it is. Can we break this up?

**Comment by daxtheduck** [comment]
@anselal this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

File: share/goodie/cheat_sheets/json/opencv-python.json (hunk 1 above)
Comment (anselal) [inline share/goodie/cheat_sheets/json/opencv-python.json]: I know, I do not like it either. I will see what I can do. Maybe I can split the code in two.

Comment (anselal) [inline share/goodie/cheat_sheets/json/opencv-python.json]: @moollaza I pushed the change I made from within the codio environment, but I do not see them at this thread nor do I get results when I am running the test server at codio. Do I have to create I new pull request for my changes ?

**Pull request merged**